lib/profile_node.stp

```diff
@@ -443,3 +443,17 @@ probe timer.profile {
         printf("\n");
 	}
 }
+
+probe timer.profile {
+    /*
+     * Since we run completely asynchronously with respect to
+     * V8's activities, we can actually catch the heap in temporarily
+     * inconsistent states, causing bad copyins and such.  Try-catch
+     * so we drop a data point in that case rather than halting.
+     */
+    try {
+        take_node_backtrace()
+    } catch {
+        next
```

Comment:
It exits the current probe: https://sourceware.org/systemtap/langref/Statement_types.html#SECTION00078000000000000000